mmp_to_musicxml/converter.py

```diff
@@ -203,16 +203,21 @@ class MMP_MusicXML_Converter:
 	
 	SPECIFIED_KEY_SIGNATURE = None
 	
-	def __init__(self, check_notes=False, key_signature=None):
+	def __init__ ( self, key_signature = None, params = None ):
 		logging.basicConfig(level=logging.DEBUG)
-		
-		if check_notes:
+
+		self.opts = params['opts']
+		self.minor = params['minor']
+
+		if self.opts.check:
```

Comment:
since `self.opts` could be `None`, I think we need a check here. something like:
`if self.opts and self.opts.check`

Reply:
Or perhaps assign `self.opts` an empty `parser.parse_args` as a default value in the class. Otherwise we'll need to check `self.opts` every time.